test/unit/arch/new_arch_first_tests.cpp

```diff
@@ -0,0 +1,71 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+
+#include <eve/arch/wide.hpp>
+
+// For a new arch.
+//
+// After getting meta things to work, this is the first set of tests that
+// need to pass.
+//
+// Comment out 1 by 1.
+
+TTS_CASE("Implement basic make function for wide<int>")
+{
+  eve::wide<int> x {[](int i, int) { return i; }};
+
+  (void)x;
+  TTS_PASS();
```

Comment:
We have TTS_COMPILE now for that, look up TTS doc.

Reply:
Thanks - I tried, doesn't seem like a good fit.

It is a good tool for sfinae things and for less error messages.
Here it doesn't help much I think